pipelines/release/tarball.groovy

```diff
@@ -208,13 +208,13 @@ def void osxTarballs(
 
         stage('publish') {
           if (publish) {
-            util.bash """
-              if ! docker ps > /dev/null 2>&1; then
-                open /Applications/Docker.app/
-                sleep 10
-              fi
-            """
-            s3PushDocker(envId)
+//            util.bash """
```

Comment:
You can just remove this code, rather than comment it out.  The old code will remain in git if we need it.